syntax = "proto3";

package hello;

service Greeter {
  rpc SayHello (HelloReq) returns (HelloReply);
}

message HelloReq {
  string name = 1;
}

message HelloReply {
  string message = 1;
}